### PR TITLE
Added sudo and amended some of the commands for removing services

### PR DESCRIPTION
**Pull request by Audarth**
Most of the commands for removing services need sudo, I've also fixed some syntax errors I missed

### CODE REVIEW

File: app/templates/INSTALL.mdown [hunk 1]
```diff
@@ -112,39 +112,41 @@ When converting a project to pm2 you should create an `ecosystem.json` file with
...
-`chkconfig -del {appname}-watch`
+`sudo chkconfig --del {appname}-watch`
 
 Then delete the scripts
 
-`rm /etc/init.d/{appname}`
+`sudo rm /etc/init.d/{appname}`
 
-`rm /etc/{appname}`
+`sudo rm -r /etc/{appname}`
```
Comment (grtjn) [inline app/templates/INSTALL.mdown]: Don't use `-r`, it is a symbolic link, not a complete directory..

File: app/templates/INSTALL.mdown [hunk 2]
```diff
@@ -112,39 +112,41 @@ When converting a project to pm2 you should create an `ecosystem.json` file with
...
 
 And for the watch script:
 
-`rm /etc/init.d/{appname}-watch`
+`sudo rm /etc/init.d/{appname}-watch`
+
+`sudo rm -r /etc/{appname}-watch`
```
Comment (grtjn) [inline app/templates/INSTALL.mdown]: same here..

**Comment by grtjn** [comment]
I happen to notice we have left etc/ in the slush template. Didn't we plan to remove it to prevent confusion?

**Comment by Audarth** [comment]
Yeah, I think it would be best

File: app/templates/INSTALL.mdown (hunk 2 above)
Comment (Audarth) [inline app/templates/INSTALL.mdown]: Hmm I had to use -r when removing the config for obi but it looks like just rm worked for alerts so it was probably just obi being a bit odd, I'll change it now

**Comment by grtjn** [comment]
Looking good, thanks, merging now!

**Pull request merged**